packages/compass-components/src/components/accordion.tsx

```diff
@@ -50,18 +50,22 @@ const buttonHintStyles = css({
 interface AccordionProps extends React.HTMLProps<HTMLButtonElement> {
   text: string | React.ReactNode;
   hintText?: string;
+  textClassName?: string;
   open?: boolean;
+  isDefaultExpanded?: boolean;
   setOpen?: (newValue: boolean) => void;
 }
 function Accordion({
   text,
   hintText,
+  textClassName,
   open: _open,
   setOpen: _setOpen,
+  isDefaultExpanded = false,
```

Comment:
Convention for controlled / uncontrolled props is to use the name of a controlled prop with a `default` or `initial` added. So `open` -> `defaultOpen` 

```suggestion
  defaultOpen = false,
```